src/Resources/config/doctrine/model/Look.orm.xml

```diff
@@ -12,6 +12,16 @@
         </id>
 
         <field name="code" column="code" type="string" unique="true"/>
+        <field name="percentageDiscount" column="percentage_discount" type="decimal" precision="10" scale="5"/>
+        <field name="enabled" column="is_enabled" type="boolean">
+            <options>
```

Comment:
Why do we need default value here?

Reply:
This is a way to have default value at fixtures. Without this, fixture should be manually changed at every app which used `v0.1.0` and then upgraded to current version. Otherwise that generated migration fail as enabled can not be null.

Oh, I missed default for `percentageDiscount`...

Reply:
Okay, I understand, but I guess no apps have yet installed this plugin. I am just a bit against default values in plugins. I don't know why... :)